### ISSUE TITLE
DApp analytics data by wallet

### ISSUE THREAD

**Issue by franckc** [OriginProtocol/origin#3767]
Currently we have no data in terms of what wallets are used: Origin Mobile app vs Metamask vs Coinbase vs IMToken, etc. This prevents us from making informed prioritization decisions when it comes to supporting those various different wallets.

We should try to detect what wallet is being used when the DApp loads and log that as a tag in Google Analytics.

**Comment by franckc** [comment]
CC @micahalcorn @matthewliu @DanielVF

**Comment by micahalcorn** [comment]
Yes, please do 👍 I've been wondering about this, but also want to be careful about making decisions based on a small amount of early usage. If we believe strongly that our app is the right experience to prioritize, then we shouldn't worry too much about which wallet vendor is generating the most traffic. Either way, more information is better.

**Comment by DanielVF** [comment]
We should probably have the origin mobile wallet easy to spot in GA, since that's a number we really care about.
